project/Build.scala

```diff
@@ -108,6 +108,8 @@ object ScalaMeterBuild extends MechaRepoBuild {
     CrossVersion.partialVersion(scalaVersion) match {
       case Some((2, 12)) => List(
         "org.scalatest" % "scalatest_2.12" % "3.0.0" % "test",
+        "junit" % "junit" % "4.12" % "test",
```

Comment:
Why the same import twice? Couldn't understand if this is some workaround or a typo.

Reply:
That was a typo (not sure how I did that!).